src/utils/githubApi.ts

```diff
@@ -103,14 +103,9 @@ export function getGitHubClient(token = ''): Octokit {
   if (!_GitHubClientCache[githubApiToken]) {
     const attrs: any = {
       auth: `token ${githubApiToken}`,
+      log: logger,
```

Comment:
Wow, have you tested this and it is _this_ easy? 😅

Reply:
Yup, that's it! :D `consola` provides all the APIs that `octokit` requires - https://octokit.github.io/rest.js/v18#logging